src/plugins/scopes/ScopesPlugin.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+
+import BasePlugin from '../BasePlugin';
+import ScopesApi from './ScopesApi';
+import type Program from '../../elements/types/Program';
+import type Node from '../../elements/Node';
+
+export default class ScopesPlugin extends BasePlugin {
```

Comment:
I would add a comment then.

> For plugins outside of cst it's not necessary to extend this class.

So if we move it to its own repo, it would stop extending it?